src/registrar/models/domain.py

```diff
@@ -1100,14 +1102,28 @@ def is_expired(self):
         now = timezone.now().date()
         return self.expiration_date < now
 
-    def state_display(self):
+    def is_expiring(self):
```

Comment:
I don't know myself so I was curious if you knew where the number came from. In that case I don't think we necessarily have to set an expiration window param in the method but let's at least make a constant within this method set to 60 that's referenced. eg
```
expiration_window = 60
...
threshold_date = now + timedelta(days=expiration_window)
```

Reply:
Will update